### ISSUE TITLE
Abnormal fetal femur/foot length ratio

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7211]
**Preferred term label:**
Abnormal fetal femur/foot length ratio

**Synonyms**

**Definition (free text, please give PubMed ID)**
The femur/foot length ratio was found to be approximately 1 throughout the age range of 14-40 weeks of gestation. 

PMID: 3292975

TODO -- get better definition, decide if term is worthwhile. Is the relevant abnormality only a reduction in the ratio or is an increased ratio also medically relevant?

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by patt1morgan** [comment]
The femur/foot length ratio nomogram appears to be a useful parameter to help differentiate fetuses that have dysplastic limb reduction from those whose limbs are short because of constitutional factors or IUGR. It can reduce the problem of an unknown gestational age and help to distinguish between fetal skeletal dysplasia and intra-uterine growth-retardation caused by other factors  PMID: 2406168
A reduction in the ratio appears to be the only measurement of concern. I could not find any articles referencing an increase in the ratio.

**Comment by pnrobinson** [comment]
thanks, @patt1morgan , adding the above definition with slight modifications.
